### PR TITLE
update Watching Mooloo

### PR DESCRIPTION
**Pull request by freakin23**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [ ] I have linked this PR to any issues that it closes.

### CODE REVIEW

File: solutions/bronze/usaco-1301.mdx [hunk 1]
```diff
@@ -78,21 +76,19 @@ public class WatchingMooloo {
 		br.close();
 		for (int i = 0; i < n; i++) { days[i] = Long.parseLong(daysInput[i]); }
 
-		Arrays.sort(days);
-
 		long lastDay = days[0];
 		long cost = k + 1;  // Start the first subscription
-		for (long d : days) {
+		for (int i = 1; i < n; i++) {
```
[the remark this answers is not in the document]
Comment (freakin23) [reply solutions/bronze/usaco-1301.mdx:81]: `days[0]` was already stored in `lastDay`  was there is no point in traversing from 0

[the remark this answers is not in the document]
Comment (freakin23) [reply solutions/bronze/usaco-1301.mdx:81]: should i restore it back to how it was?

[the remark this answers is not in the document]
Comment (SansPapyrus683) [reply solutions/bronze/usaco-1301.mdx:81]: yeah

[the remark this answers is not in the document]
Comment (freakin23) [reply solutions/bronze/usaco-1301.mdx]: done

File: solutions/bronze/usaco-1301.mdx [hunk 2]
```diff
@@ -38,21 +38,19 @@ int main() {
...
-
 	long long last_day = days[0];
 	long long cost = k + 1;  // Start the first subscription
-	for (long long d : days) {
+	for (int i = 0; i < n; i++) {
 		// Should Bessie extend the most recent subscription?
-		if (d - last_day < k + 1) {
-			cost += d - last_day;
+		if (days[i] - last_day < k + 1) {
+			cost += days[i] - last_day;
```
Comment (SansPapyrus683) [inline solutions/bronze/usaco-1301.mdx:46]: not changed

Comment (freakin23) [reply solutions/bronze/usaco-1301.mdx]: changed.
i thought that by just  changing the initial value to 0 would be enough.

File: solutions/bronze/usaco-1301.mdx [hunk 3]
```diff
@@ -78,8 +76,6 @@ public class WatchingMooloo {
 		br.close();
 		for (int i = 0; i < n; i++) { days[i] = Long.parseLong(daysInput[i]); }
 
-		Arrays.sort(days);
```
Comment (SansPapyrus683) [inline solutions/bronze/usaco-1301.mdx:81]: wait why was this removed again

Comment (freakin23) [reply solutions/bronze/usaco-1301.mdx:81]: the elements are already in sorted order as per stated by the question

**Review by SansPapyrus683 — APPROVED** [review]
